### PR TITLE
Add setup.py file

### LINKED ISSUE #2: no setup.py in bundle
The activity in github does not have a setup.py file.

### PR DESCRIPTION
**Pull request by nswarup14**
Fixes #2 
Thanks

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from sugar3.activity import bundlebuilder
+
+if __name__ == "__main__":
+    bundlebuilder.start()
```
Comment (rhl-bthr) [inline setup.py]: Missing newline

**Review by rhl-bthr — CHANGES_REQUESTED** [review]
Thanks, reviewed

**Comment by nswarup14** [comment]
@pro-panda 
Added the newline
Please review, thanks

File: setup.py (hunk 1 above)
Comment (nswarup14) [reply setup.py]: Done

**Comment by rhl-bthr** [comment]
Thanks

**Pull request merged**